tests/test_inputs.py

```diff
@@ -55,18 +56,19 @@ def test_pdf_cut_n_pages(numb_pages: int):
     )
     assert input_obj.count_doc_pages() == numb_pages
 
-    # Each page in the PDF has a unique (and increasing) /Content /Length.
-    # We use this to make sure we have the correct pages
-    cut_pdf = pikepdf.open(input_obj.file_object)
-    with pikepdf.open(
-        FILE_TYPES_DIR / "pdf" / f"multipage_cut-{numb_pages}.pdf"
-    ) as pdf:
-        for idx, page in enumerate(pdf.pages):
-            assert (
-                page["/Contents"]["/Length"]
-                == cut_pdf.pages[idx]["/Contents"]["/Length"]
-            )
+    # Currently the least verbose way of comparing pages with pypdfium2
+    # I.e. each page is read & rendered as a rasterized image. These images are then compared as raw byte sequences.
+    cut_pdf = pdfium.PdfDocument(input_obj.file_object)
+    pdf = pdfium.PdfDocument(FILE_TYPES_DIR / "pdf" / f"multipage_cut-{numb_pages}.pdf")
+    for idx in range(len(pdf)):
+        pdf_page = pdf.get_page(idx)
```

Comment:
iterator maybe ?

Reply:
There's two of them, this keeps the syntax a tad more legible, I'd rather leave it as is.

Reply:
okok